src/modules/displayMovies.js

```diff
@@ -0,0 +1,46 @@
+import { updateLikes, setLike } from './likeFunctionality.js';
+
+const displayMovies = async (movieList, appId, section) => {
+  const movieSection = document.querySelector(`#${section}-section`);
+  movieSection.innerHTML = '';
+  movieList.forEach((item) => {
+    const mainDiv = document.createElement('div');
+    mainDiv.className = 'movie-wrapper';
+    mainDiv.id = `${section}${item.id}`;
+    const details = document.createElement('div');
+    details.className = 'movie-wrapper__title';
+    const img = document.createElement('img');
+    img.className = 'movie-wrapper__img';
+    img.src = item.image.medium;
+    img.id = `img_${item.id}`;
+    const name = document.createElement('h3');
+    name.innerHTML = `${item.name}`;
+    name.id = `name${item.id}`;
+    const likes = document.createElement('div');
+    likes.className = 'movie-wrapper__likes';
+    const span = document.createElement('span');
+    const like = document.createElement('i');
+    like.className = 'fas fa-heart';
+    likes.append(like, span);
+    details.append(name, likes);
+    const commentButton = document.createElement('button');
+    commentButton.type = 'button';
+    commentButton.innerHTML = 'Comments';
+    commentButton.className = 'movie-wrapper__comment-button';
+    mainDiv.append(img, details, commentButton);
+    const reserveButton = document.createElement('button');
+    reserveButton.type = 'button';
+    reserveButton.innerHTML = 'Reservations';
+    reserveButton.className = 'movie-wrapper__comment-button';
+    mainDiv.append(reserveButton);
+    movieSection.appendChild(mainDiv);
+    like.addEventListener('click', () => {
```

Comment:
- [ ] The pop-up shows up after clicking on the like button. Please ensure that after clicking the like button, only the number of likes is increased without displaying the pop-up.